### PR TITLE
docs : Add User Story template to project #40

### LINKED ISSUE #40: Add User Story template to project
**Is your feature request related to a problem? Please describe.**
User story template is missing and this can be great for well-defined issues and guiding contributors.

Link to raw template on backend repo: https://raw.githubusercontent.com/anitab-org/mentorship-backend/develop/.github/ISSUE_TEMPLATE/user-story.md

**Describe the solution you'd like**

Add User Story template to `.github/ISSUE_TEMPLATE` folder

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by Sanika1999**
### Description
Include a summary of the change and relevant motivation/context. List any dependencies that are required for this change.

Fixes #40 

### Flutter Channel:
- [ ] I have used the Flutter Beta channel on my local machine 

### Type of Change:
**Delete irrelevant options.**

- Code
- Quality Assurance
- User Interface
- Outreach
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/user-story.md
Comment (bartekpacia) [inline .github/ISSUE_TEMPLATE/user-story.md]: Will a typical, non tech-savyy user know what to put here? (mocks)

File: .github/ISSUE_TEMPLATE/user-story.md
Comment (bartekpacia) [inline .github/ISSUE_TEMPLATE/user-story.md]: Also, how can a user know how much time adding a particular feature would take? I'd remove this or change to `[INSERT NUMBER HERE] hours or [REMOVE THIS IF YOU DON'T KNOW]`.

**Review by bartekpacia — CHANGES_REQUESTED** [review]
Looks quite good. Please take a look at the comments and update your PR description. You didn't change anything in the template.

File: .github/ISSUE_TEMPLATE/user-story.md
Comment (isabelcosta) [reply .github/ISSUE_TEMPLATE/user-story.md]: good point @bartekpacia . estimation in time is subjective as well, I probably cannot estimate issues that involve changing flutter code. For this PR I am fine with either, for the future this is a good issue to bring to @anitab-org/content-team

File: .github/ISSUE_TEMPLATE/user-story.md
Comment (isabelcosta) [reply .github/ISSUE_TEMPLATE/user-story.md]: This has the description below `[INSERT RELEVANT PNG FILE]`. What do you suggest here @bartekpacia so @Sanika1999 knows what is the change requested.

File: .github/ISSUE_TEMPLATE/user-story.md
Comment (aa0501) [reply .github/ISSUE_TEMPLATE/user-story.md]: +1 @bartekpacia 
I myself had trouble estimating time periods; it is just quite difficult to say how much this fix/enhancement would take (specially if it requires some sort of research).

File: .github/ISSUE_TEMPLATE/user-story.md
Comment (bartekpacia) [reply .github/ISSUE_TEMPLATE/user-story.md]: I'd change "mocks" to simply "screenshots of the issue", it's more clear to a typical person in my opinion.

**Review by isabelcosta — APPROVED** [review]
Seems good to me :) Let's wait for other maintainers review.
Thank you for your contribution @Sanika1999 🎉

**Comment by Sanika1999** [comment]
> Seems good to me :) Let's wait for other maintainers review.
> Thank you for your contribution @Sanika1999 🎉

thank you @isabelcosta  OK  👍

**Review by bartekpacia — APPROVED** [review]

**Pull request merged**
